### PR TITLE
More run command tests

### PR DESCRIPTION
**Pull request by gabriel**
Includes testing SIGTERM timeout

### CODE REVIEW

**Comment by gabriel** [comment]
@keybase/updater-hackers

File: command_test.go [hunk 1]
```diff
@@ -125,14 +149,21 @@ func TestTimeoutProcessKilled(t *testing.T) {
 	assert.NotNil(t, perr, "Should have errored killing since killing non-existant process should error")
 }
 
+func TestRunCommandNoExit(t *testing.T) {
+	path := filepath.Join(os.Getenv("GOPATH"), "bin", "test")
+	_, err := RunCommand(path, []string{}, 10*time.Millisecond, log)
+	require.Error(t, err)
+	t.Logf("Error: %s", err)
```
Comment (jzila) [inline command_test.go]: preferably use `EqualError` here.

File: command_test.go [hunk 2]
```diff
@@ -94,6 +104,20 @@ func TestRunJSONCommand(t *testing.T) {
...
+	require.Error(t, err)
+}
+
+func TestRunJSONCommandInvalidObject(t *testing.T) {
+	// Valid JSON, but not the right object
+	validJSON := `{"stringVar": true}`
+	var testValOut testObj
+	err := RunJSONCommand("echo", []string{validJSON}, &testValOut, time.Second, log)
+	require.Error(t, err)
+	t.Logf("Error: %s", err)
```
Comment (jzila) [inline command_test.go]: `EqualError` here please

**Comment by jzila** [comment]
LGTM after nits

**Pull request merged**